samples/samples-csharpscript/product.csx

```diff
@@ -0,0 +1,63 @@
+public class Product
```

Comment:
Does this actually need to be a .csx or can it just be a .cs? It doesn't seem to use any scripting functionality. 

I'd also suggest putting it in a "Common" folder

Reply:
It's an example of how to use scripting files and include them, if you do .cs the way we include it in the run.csx file is using statement.